src/GiftedChat.tsx

```diff
@@ -583,12 +583,8 @@ class GiftedChat<TMessage extends IMessage = IMessage> extends React.Component<
     )
   }
 
-  safeAreaSupport = (bottomOffset: number) => {
-    return bottomOffset === this._bottomOffset
-      ? this.getBottomOffset()
-        ? this.getBottomOffset()
-        : getBottomSpace()
-      : bottomOffset
+  safeAreaSupport = (bottomOffset?: number) => {
+    return bottomOffset != null ? bottomOffset : getBottomSpace()
```

Comment:
```suggestion
    return bottomOffset !== null ? bottomOffset : getBottomSpace()
```

Reply:
@xcarpentier I use `!=` to handle if bottomOffset is `undefined`

Reply:
?